contracts/StablecoinConverter.sol

```diff
@@ -244,6 +251,22 @@ contract StablecoinConverter is EpochTokenLocker {
         );
     }
 
+    function evaluateDisregardedUtility(Order memory order, address user) internal view returns(uint128) {
+        // |disregardedUtility| = maxUtility - actualUtility
```

Comment:
If this is is the definition, shouldn't we just call `evaluateUtility(remainingBuyAmount, order) - evaluateUtility(executedBuyAmount, order)`?

Reply:
Not quite, since utility involves a slightly different version of executedSellAmount and this is also no longer the formula being used. Thanks for pointing this out, I will update it!